server/lib/validation/languages/yue.js

```diff
@@ -0,0 +1,34 @@
+// Minimum of words that qualify as a sentence.
```

Comment:
minor: could you please change the comment here as it's now number of characters and not words?